src/Aspire.Hosting/build/Aspire.Hosting.targets

```diff
@@ -32,33 +36,27 @@
     <ItemGroup>
       <ServiceMetadataSource Update="@(ServiceMetadataSource)">
         <Source>
-        <![CDATA[using System%3B
+        <![CDATA[namespace Projects%3B
 
-namespace ]]>%(Namespace)<![CDATA[%3B
-
-[System.Diagnostics.DebuggerDisplay("Type = {GetType().Name,nq}, ProjectPath = {ProjectPath}")]
-public class ]]>%(ClassName)<![CDATA[ : IServiceMetadata
+[global::System.Diagnostics.DebuggerDisplay("Type = {GetType().Name,nq}, ProjectPath = {ProjectPath}")]
+public class ]]>%(ClassName)<![CDATA[ : global::Aspire.Hosting.IServiceMetadata
```

Comment:
NIT: Should we also generate an `ExcludeFromCodeCoverage` attribute? or altenarively a `Generated` one?

Reply:
I've opened https://github.com/dotnet/aspire/issues/1792 for this.